packages/libsql-client/src/http.ts

```diff
@@ -24,9 +24,12 @@ import {
     getIsSchemaDatabase,
     waitForLastMigrationJobToFinish,
 } from "./migrations.js";
+import promiseLimit from "promise-limit";
 
 export * from "@libsql/core/api";
 
+const limit = promiseLimit<ResultSet>(1);
```

Comment:
I don't think we should use a global one. We should go with a limit per client.
It protects us less, but gives us more predictability.

We should allow users to override the limit value.
I suggest starting with something low, like 20, and exposing a field at the client config object allow users to override it - maybe disable it?

Reply:
I just deployed a change that implements this:

```js
export const turso = createClient({
  url: process.env.TURSO_DATABASE_URL,
  authToken: process.env.TURSO_AUTH_TOKEN,
  concurrency: 10 // defaults to 20 and undefined means there's no limit
});
```

This limit is now per client.